File: PPA/Dailycodes/statements/FlowControlStmt/08july2020/programs/calculator.c
#include<stdio.h>
void main(){
	int x,y;
	int choice;
	printf("Enter two values\n");
	scanf("%d %d",&x, &y);
	
	printf("Enter your choice\n");
	printf("1 : Addition\n");
	printf("2 : substraction\n");
	printf("3 : multiplication\n");
	printf("4 : Division\n");
	scanf("%d",&choice);
	
	switch(choice){
		case 1:
			printf("Addition = %d\n",x+y);
			break;
			
		case 2:
			printf("Substraction = %d\n",x-y);
			break;
			
		case 3:
			printf("multiplication = %d\n",x*y);
			break;
		
		case 4:
			printf("Division = %d\n",x/y);
			break;
			
		default:
			printf("Enter correct choice\n");
			break;	
	}
}

File: Java/DailyFlash/30.07.2020/programs/Calculator.java
/*
write a java program to get two numbers and one operator
from user and perform calculator operations(+,-,/,*,%) 
on numbers using switch case.
*/

import java.util.*;
class Calculator{
	public static void main(String[]  arg){

	
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter First nnumber:");
		int num1=sc.nextInt();
		
		System.out.println("Enter First nnumber:");
		int num2=sc.nextInt();
		
		System.out.println("Enter operator:");
		String str=sc.next();
		char op=str.charAt(0);
		
		
		switch(op){
			case '+':
				System.out.println("Addition is = "+(num1+num2));
				break;
				
			case '-':
				System.out.println("substraction is = "+(num1-num2));
				break;
				
			case '*':
				System.out.println("Multiplication is = "+(num1*num2));
				break;
				
			case '/':
				System.out.println("Division is = "+(num1/num2));
				break;
				
			case '%':
				System.out.println("Modulus is = "+(num1%num2));
				break;
				
			default:
				System.out.println("Enter valid data");
				break;
			
		}
	}
}
